edu.cuny.hunter.mylyngit.core/src/edu/cuny/hunter/mylyngit/core/utils/Util.java

```diff
@@ -147,6 +148,7 @@ private static File findEvaluationPropertiesFile(IJavaProject project, String fi
 	/**
 	 * Create CompilationUnit from ICompilationUnit.
 	 */
+	@SuppressWarnings("deprecation")
```

Comment:
Same here.

Reply:
The field AST.JLS8 is deprecated

Reply:
Then use the non-deprecated one rather than suppressing this warning. Thanks.

Reply:
I tried AST.JLS11, but it could fail building.

Reply:
What is the error?

Reply:
If you change it, we can see it fail on travis.

Reply:
https://travis-ci.com/ponder-lab/Logging-Level-Evolution-Plugin/builds/107693082

Reply:
It is compilation failure, but I don't see it on eclipse.

Reply:
So it only fails in travis?

Reply:
Yes.

Reply:
What happens if you choose JLS10?

Reply:
Do you get a warning in Eclipse still?

Reply:
This is the version of Eclipse the maven build is using: https://help.eclipse.org/2018-09/topic/org.eclipse.jdt.doc.isv/reference/api/org/eclipse/jdt/core/dom/AST.html#JLS10

Reply:
I tried JLS10, but got the same warning.

Reply:
I am using eclipse 2018-12. https://help.eclipse.org/2018-12/index.jsp?topic=%2Forg.eclipse.jdt.doc.isv%2Freference%2Fapi%2Forg%2Feclipse%2Fjdt%2Fcore%2Fdom%2FAST.html&anchor=JLS11

Reply:
Should I change my eclipse version? I don't think it is necessary.

Reply:
The same warning in Eclipse or on travis?

Reply:
In eclipse. I did not push that change.

Reply:
OK, change it JLS-11. I will set the Eclipse version to 2019-12 in the `pom.xml`.

Reply:
OK.

Reply:
Hm. Looks like the tests are failing with the newer Eclipse version.

Reply:
Is the warning caused by JDK version?

Reply:
OK, let's do this. Keep the warning suppression but change it https://help.eclipse.org/2018-09/topic/org.eclipse.jdt.doc.isv/reference/api/org/eclipse/jdt/core/dom/AST.html#JLS10.

Reply:
I see.